YACS-RCOS/yacs.n
# Fixed csv time parse bug, and added updated spring 2021 csv

**Issue**

Closes #309 

**Test Procedure**

Show procedure to test functionality with a clear procedure

*Example:*
> 1. Compile a new Spring 2021 CSV
> 2. Upload the CSV in the admin panel of your test enviroment
> 3. Check that ECSE 2050 & ECSE 2100 display correctly
